test/plates-model/crust.test.ts

```diff
@@ -14,11 +14,19 @@ describe("Crust model", () => {
       { rock: Rock.Gabbro, thickness: 0.10500000000000001 }
     ]);
 
+    // Continental shelf -> no continental sediments.
     crust = new Crust("continent", 0.5);
     expect(crust.rockLayers).toEqual([
-      { rock: Rock.ContinentalSediment, thickness: 0.05 },
       { rock: Rock.Limestone, thickness: 0.125 },
-      { rock: Rock.Granite, thickness: 0.325 }
+      { rock: Rock.Granite, thickness: 0.375 }
+    ]);
+
+    console.log(BASE_CONTINENTAL_CRUST_THICKNESS);
```

Comment:
Did you intend to leave this `console.log` in the test code?

Reply:
No, I just removed it. Thanks!